Resources/views/frontend/checkout/ajax_cart.tpl

```diff
@@ -7,6 +7,16 @@
             <div class="apple-pay--container apple-pay--container--ajax-cart">
                 {include 'frontend/plugins/payment/mollie_applepay_direct.tpl' }
             </div>
+            <script type="text/javascript">
+                // This needs to be inline to force the script on offcanvas reload
+                var applePayContainer = $('.apple-pay--container--ajax-cart');
```

Comment:
can we move this to javascript files please?

Reply:
Sadly not, since there seems to be some issue with safari.

Reply:
? all apple pay javascript code is placed in js files? that makes no sense?

Reply:
as discussed, we need to do the initApplePay functionality inline.